src/features/click-to-load.js

```diff
@@ -831,6 +835,21 @@ function resizeElementToMatch (sourceElement, targetElement) {
     for (const key of stylesToCopy) {
         targetElement.style[key] = computedStyle[key]
     }
+    // Sometimes, after copying 'top', 'bottom', 'left', 'right' from an absolute
+    // positioned element, the 'inset' property is automatically set to a value
+    // trying to replicate that positioning. But that often is not accurate, so we
+    // make sure to reset it back to 'auto' here.
+    targetElement.style.inset = 'auto'
```

Comment:
As mentioned in MatterMost, this `inset = 'auto'` line is causing placeholders to be incorrectly positioned on a few websites. Suggest we just remove this for now.

Reply:
👍